src/athens/sandbox/api.clj

```diff
@@ -1,25 +1,94 @@
 (ns ^{:doc "End-user API expected to be exposed to learners in the REPL"}
```

Comment:
Are we scrapping everything in the Athens sandbox and just going with the ClojureFam route? If so, may make sense to just comment or remove any files in here.

Reply:
Hmm, I agree.

I think we're going to need the part about datascript DB loading later when writing Roam db koans. But we can easily put that back at a later point.

Yup, let's remove it.